sw/device/silicon_creator/rom/e2e/rom_ext_upgrade_interrupt/BUILD

```diff
@@ -87,22 +88,21 @@ otp_json(
 ]
 
 [
-    opentitan_functest(
+    opentitan_test(
         name = "rom_ext_upgrade_interrupt_{}".format(lc_state),
         srcs = ["rom_ext_upgrade_interrupt.c"],
         cw310 = cw310_params(
             bitstream = ":bitstream_rom_ext_upgrade_interrupt_{}".format(lc_state),
-            clear_bitstream = True,
```

Comment:
I am not sure why the test previoulsy required `clear_bitstream`? We don't have it in the new rules but that could be added back if necessary.

Reply:
I tested locally without this flag and got the test to pass, but I think it may have been there to make the initial test stable. I have added back the clear-bitstream arg. 

I will chat with @cfrantz to see what he recommends re. adding the `clear_bitstream` arg to `cw310_arams`.